docs/user/security/securing-communications/elasticsearch-mutual-tls.asciidoc

```diff
@@ -0,0 +1,120 @@
+[role="xpack"]
+[[elasticsearch-mutual-tls]]
+=== Mutual TLS authentication between {kib} and {es}
+++++
+<titleabbrev>Mutual TLS with {es}</titleabbrev>
+++++
+
+In a standard Transport Layer Security (TLS/SSL) configuration, the server presents a signed certificate to authenticate itself to the
+client. In a mutual TLS configuration, the client also presents a signed certificate to authenticate itself to the server.
+
+When {security} is enabled on your cluster, each request that {kib} makes to {es} must be authenticated. Most requests made through {kib} to
+{es} are authenticated by using the credentials of the logged-in user. There are, however, a few internal requests that the {kib} server
+needs to make to the {es} cluster. For this reason, you must configure credentials for the {kib} server to use for those requests.
+
+If {kib} has `elasticsearch.username` and `elasticsearch.password` configured, it will attempt to use these authenticate to {es} via the
+{ref}/configuring-native-realm.html[Native realm]. However, {kib} also supports mutual TLS authentication with {es} via a
```

Comment:
The Native realm link goes to a deleted page.  Should link to {ref}/security-settings.html#ref-native-settings, I think.

Reply:
Oh, whoops. I was originally looking at the docs for 7.5, where it doesn't show as a deleted page.

I'll change this to [`{ref}/native-realm.html#native-realm-configuration`](https://www.elastic.co/guide/en/elasticsearch/reference/master/native-realm.html#native-realm-configuration).